assets/charts/control-plane/kube-apiserver/templates/kube-apiserver-secret.yaml

```diff
@@ -2,6 +2,7 @@
   apiserver.key: "{{ .Values.apiserver.apiserverKey }}"
   apiserver.crt: "{{ .Values.apiserver.apiserverCert }}"
   service-account.pub: "{{ .Values.apiserver.serviceAccountPub }}"
```

Comment:
We no longer need a public key in `kube-apiserver` chart, we can now use private key for `--service-account-key-file` flag.